components/UserProvider.js

```diff
@@ -72,7 +72,6 @@ class UserProvider extends React.Component {
 
   logout = async () => {
     removeFromLocalStorage(LOCAL_STORAGE_KEYS.ACCESS_TOKEN);
-    removeFromLocalStorage(LOCAL_STORAGE_KEYS.LOGGED_IN_USER);
```

Comment:
Leftover from https://github.com/opencollective/opencollective-frontend/pull/6387, this one was not doing anything